.github/workflows/link-validator.yml

```diff
@@ -0,0 +1,37 @@
+name: Link Validator
+
+on:
+  pull_request:
```

Comment:
Do we really want it on every pull request? That sounds like a lot of link validating even though links are seldom changed/introduced.